packages/paste-website/src/components/popular-components-and-patterns/PopularSectionHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/core/box';
+
+const PopularSectionHeader: React.FC = () => {
+  return (
+    <Box
+      as="h4"
+      color="colorTextWeak"
+      fontWeight="fontWeightBold"
+      marginBottom="space110"
+      marginTop="space0"
+      position="relative"
+      _after={{
+        borderTopColor: 'colorBorderLight',
+        borderTopStyle: 'solid',
+        borderTopWidth: 'borderWidth10',
+        content: `""`,
+        display: 'block',
+        height: '1px',
+        marginLeft: 'space160',
+        position: 'absolute',
+        top: '50%',
+        width: 'calc(100% - 3.75rem)', // 100% - space160
```

Comment:
Out of curiosity, could we use `useTheme()` in this component to pass the token directly in this calc?

Reply:
Oh yeah. Good call. We can do that.